web_advanced_filter/README.rst

```diff
@@ -0,0 +1,101 @@
+================
+Advanced filters
+================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fweb-lightgray.png?logo=github
+    :target: https://github.com/OCA/web/tree/12.0/web_advanced_filter
+    :alt: OCA/web
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/web-12-0/web-12-0-web_advanced_filter
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/162/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This addon allows users to apply set operations on filters: Remove or add
+certain ids from/to a selection, but also to remove or add another filter's
+outcome from/to a filter. This can be stacked, so the filter domain can be
+arbitrarily complicated.
+
+The math is hidden from the user as far as possible, in the hope it's still
+user friendly.
+
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+After this addon is installed, every search view shows a new menu `Advanced
+filters`. Here the set operations can be applied as necessary.
+
+Note the menu is only visible when there's search criteria or some record
+selected.
+
+Known issues / Roadmap
+======================
+
+* client side tests
+* consider renaming all instances of `Favorites` in core to `Filters` for consistency
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/web/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/web/issues/new?body=module:%20web_advanced_filter%0Aversion:%2012.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Do not contact contributors directly about support or help with technical issues.
+
+Credits
+=======
+
+Authors
+~~~~~~~
+
+* Therp BV
+
+Contributors
+~~~~~~~~~~~~
+
+* Holger Brunn <hbrunn@therp.nl>
```

Comment:
```suggestion
* Holger Brunn <mail@hunki-enterprises.com>
```

Reply:
@janverb if you see this early enough, I'd be thankful if you add this change

Reply:
@hbrunn maybe you can make a PR that changes your email address over the whole repo. Normally we have a commit / PR per module, but I have seen at least one other example of such a PR.

Anyway thanks for the approval and the merge,